dask_expr/tests/test_concat.py

```diff
@@ -65,3 +65,15 @@ def test_concat_one_no_columns(df, pdf):
     result = concat([df, df[[]]])
     expected = pd.concat([pdf, pdf[[]]])
     assert_eq(result, expected)
+
+
+def test_concat_simplify(pdf, df):
+    pdf2 = pdf.copy()
+    pdf2["z"] = 1
+    df2 = from_pandas(pdf2)
+    q = concat([df, df2])[["z", "x"]]
+    result = q.simplify()
+    expected = concat([df[["x"]], df2[["x", "z"]]])._simplify_down()[["z", "x"]]
```

Comment:
Does this not work with `simplify` instead of `_simplify_down`?

Reply:
Simplify would remove the projection in the end to, I just want to push it down into StackPartition and keep everything else as is

Reply:
Okay, so you are saying `concat([df[["x"]], df2[["x", "z"]]])._simplify_down()` is **not** the same as `concat([df[["x"]], df2[["x", "z"]]]).simplify()`? Interesting.

Reply:
No sorry my comment was wrong, this should yield the same result, otherwise something else is buggy.

Initially, I had it at then end, which would have produced something different, but doing it in the middle should work. Sorry for the confusion

Reply:
Okay cool. That makes me feel better :)